vault/resource_pki_secret_backend_cert.go

```diff
@@ -167,6 +176,12 @@ func pkiSecretBackendCertCreate(d *schema.ResourceData, meta interface{}) error
 		ipSans = append(ipSans, iIpSan.(string))
 	}
 
+	iURISans := d.Get("uri_sans").([]interface{})
```

Comment:
Could we simplify this code if we did something more like:
```
if raw, ok := d.GetOk("uri_sans"); ok {
    data["uri_sans"] = raw
}
```
Or is there a downside to that?

Reply:
Hi @tyrannosaurus-becks,
    Same idea as the last comment. I jacked most of this code from another PR. https://github.com/terraform-providers/terraform-provider-vault/pull/373/files#diff-318e4d5fc8ea223f57928c837b6ea671R193. It smells like your block would be equivalent, but maybe it makes sense to keep the two related blocks to use the same form? Not sure your opinion there.